ert3/evaluator/_evaluator.py

```diff
@@ -1,25 +1,98 @@
 import asyncio
-import os
 import pathlib
 import shutil
 from collections import defaultdict
 from pathlib import Path
-from typing import Dict, Any, Tuple, List
+from typing import Coroutine, Dict, Any, Tuple, List
 
 
 import cloudpickle
-from pydantic import FilePath
 
 import ert3
 from ert_shared.ensemble_evaluator.config import EvaluatorServerConfig
 from ert_shared.ensemble_evaluator.evaluator import EnsembleEvaluator
 from ert_shared.ensemble_evaluator.prefect_ensemble import PrefectEnsemble
-
-from ert3.config import EnsembleConfig, StagesConfig, Step
-from ert3.data import EnsembleRecord, MultiEnsembleRecord, RecordTransmitter, Record
+from ert3.config import (
+    EnsembleConfig,
+    StagesConfig,
+    TransportableCommand,
+    Unix,
+    Function,
+    Step,
+)
+from ert3.data import (
+    EnsembleRecord,
+    MultiEnsembleRecord,
+    RealisationsToRecordToTransmitter,
+    RecordTransmitter,
+    Record,
+    SharedDiskRecordTransmitter,
+)
 
 _EVTYPE_SNAPSHOT_STOPPED = "Stopped"
 _EVTYPE_SNAPSHOT_FAILED = "Failed"
+_MY_STORAGE_DIRECTORY = ".my_storage"
```

Comment:
Should we leak storage stuff into this module? And what is that path? 😅 Maybe reconsider the naming here.